spec/acceptance/cisco_aci_fvaepg/base_rspec.rb

```diff
@@ -23,17 +23,36 @@
   context 'Base acceptance test' do
     basedata_fname = File.dirname(__FILE__) + '/fvaepg_basedata.pp'
     basedata = File.read(basedata_fname)
+    it 'fvaepg Testcase Setup' do
+      manifest = "$override_ensure = absent\n$override_descr = undef\n" + basedata
+      apply_manifest(manifest)
+    end
     it 'Create (ensure = present)' do
       manifest = "$override_ensure = present\n$override_descr = undef\n" + basedata
-      apply_manifest(manifest)
+      output, error = apply_manifest(manifest)
```

Comment:
Lot of common code here.  Can you turn this into a procedure?

Reply:
done.